pages/_app.tsx

```diff
@@ -22,8 +22,10 @@ const meta = {
 
 export default function App({ Component, pageProps, router }: AppProps) {
   useEffect(() => {
-    const handleStart = () => {
-      NProgress.start();
+    const handleStart = (_: string, { shallow }: any) => {
```

Comment:
```suggestion
    const handleStart = (_: string, { shallow }: { shallow: boolean }) => {
```

Make sure to not use `any`.

Reply:
Thanks, updated!